### PR TITLE
Update script

### PR DESCRIPTION
**Pull request by vvvakulenko**
@istoimenova Please review

### CODE REVIEW

File: test_scripts/Polices/External_UCS_Informing_HMI/017_ATF_N_GetListofPermissions_empty_due_to_missing_entityID_OnAppPermissionChange.lua [hunk 1]
```diff
@@ -81,7 +82,7 @@ function Test:Precondition_PTU_and_OnAppPermissionConsent_AllParams_Valid()
           {
             appID = self.applications[config.application1.registerAppInterfaceParams.appName],
             consentedFunctions = {
-              { allowed = true, id = 156072572, name = "Location-1"},
+              { allowed = true, id = 156072572, name = "Location"},
```
Comment (istoimenova) [inline test_scripts/Polices/External_UCS_Informing_HMI/017_ATF_N_GetListofPermissions_empty_due_to_missing_entityID_OnAppPermissionChange.lua]: @vvvakulenko, please return "Location-1"

File: test_scripts/Polices/External_UCS_Informing_HMI/017_ATF_N_GetListofPermissions_empty_due_to_missing_entityID_OnAppPermissionChange.lua [hunk 2]
```diff
@@ -103,7 +104,7 @@ end
 --[[ Test ]]
 commonFunctions:newTestCasesGroup("Test")
 
-function Test:TestStep_GetListofPermissions_entityType_missing()
+function Test:TestStep_GetListofPermissions_entityID_missing()
   local RequestIdListOfPermissions = self.hmiConnection:SendRequest("SDL.GetListOfPermissions", {appID = self.applications[config.application1.registerAppInterfaceParams.appName]})
   
   EXPECT_HMIRESPONSE(RequestIdListOfPermissions, {
```
Comment (istoimenova) [inline test_scripts/Polices/External_UCS_Informing_HMI/017_ATF_N_GetListofPermissions_empty_due_to_missing_entityID_OnAppPermissionChange.lua]: @vvvakulenko, on line 113 "Location-1", should be only "Location"

Comment (vvvakulenko) [inline test_scripts/Polices/External_UCS_Informing_HMI/017_ATF_N_GetListofPermissions_empty_due_to_missing_entityID_OnAppPermissionChange.lua]: @istoimenova please see updatet in commit ea2a524d288dd0d792998c185777040e0e8192a6

Comment (vvvakulenko) [inline test_scripts/Polices/External_UCS_Informing_HMI/017_ATF_N_GetListofPermissions_empty_due_to_missing_entityID_OnAppPermissionChange.lua]: @istoimenova please see updated in ea2a524d288dd0d792998c185777040e0e8192a6

**Pull request merged**